test/conformance/kuttl/validate/clusterpolicy/standard/enforce/configmap-context-lookup/README.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+The configmap context lookup uses informer's cache internally, the background processing should use the same to resolve configmap context without crashing Kyverno.
+
+## Expected Behavior
+
+Policy report is generated properly.
```

Comment:
See previous comment. If this is the expectation we should be checking the final resource and not the intermediary one. The indirect behavior is that the policy is created successfully and this is important to state.